.gitignore

```diff
@@ -1,2 +1,4 @@
 .bin/
 build/
+build-nodejs-18
```

Comment:
the `create.sh` script generates them under the build, build-nodejs-18 and build-nodejs-16 directory. Later on in the github workflow the upload action, uploads them under the same path so, after it downloads them under the build, build-nodejs-18 and build-nodejs-16 accordingly. It would be nice to have everything under the build directory in terms of less changes each time we introduce a new node version, but this would require on the current pr changes to the `create.sh` script which I would avoid ATM as it would need another iteration of code reviews. I agree with this suggestion as a future improvement.

Reply:
Sure, that's reasonable.